http_cache/expiration.rst

```diff
@@ -26,14 +26,22 @@ is used to specify many different cache directives::
 
     // sets the number of seconds after which the response
     // should no longer be considered fresh by shared caches
-    $response->setSharedMaxAge(600);
+    $response->setPublic();
+    $response->setMaxAge(600);
 
 The ``Cache-Control`` header would take on the following format (it may have
 additional directives):
 
 .. code-block:: text
 
-    Cache-Control: public, s-maxage=600
+    Cache-Control: public, maxage=600
+
+.. note::
+
+    Using the ``setSharedMaxAge()`` may seem equivalent to using both ``setPublic()``
+    and ``setMaxAge()`` methods. However, their behavior is different in a
```

Comment:
My suggestion would be to refer to https://tools.ietf.org/html/rfc7234#section-4.2.4:

"However, the `setSharedMaxAge()` method and the corresponsing `s-maxage` setting have the additional implication of prohibiting a cache to use a stale response in particular error scenarios. See RFC 7234 Section 4.2.4 for details."

(Does "stale" make sense in the context of this page, is it a term we have used or defined before?)

Reply:
Thanks for your review! I added this while merging the pull request.